lib/_utils/constants/routes.dart

```diff
@@ -16,7 +16,8 @@ class RouteNames {
 final publicPages = [
   GetPage(name: '/', page: () => const Welcome()),
   GetPage(name: RouteNames.loginPage, page: () => const Login()),
-  GetPage(name: RouteNames.dashboard, page: () => const Dashboard()),
+  GetPage(
+      name: RouteNames.dashboard, page: () => const CompleteProfileScreen()),
```

Comment:
how can be route name "dashboard" and screen name is profile screen.. and its not actually a profile screen it's a signup or register screen.